.github/workflows/build.yml

```diff
@@ -97,29 +92,26 @@ jobs:
   static-analysis:
     name: Static analysis
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        #runs on the lowest and highest supported versions (change this to 8.4 once psalm supports it)
+        php: [8.1, 8.3]
```

Comment:
Just to confirm, I think this means we're now running with symfony 6 (on 8.1) and symfony 7 (on 8.3) but not with symfony 5. Is it worth adding symfony5 to the matrix as well, or is that not worth it for the cost / benefit?

Reply:
Symfony 5.4 is now security fixes only, so I think it is not worth it